### DISCUSSION TITLE
WireGuard на роутере (keenetik)

### DISCUSSION THREAD

**Discussion by VitArts** [xtrime-ru/antizapret-vpn-docker#52] [Q&A] [answered]
Привет, реально ли настроить wireguard на роутере, он не позволяет получать маршруты от удаленной стороны? И ограничен чем openvpn в настройках?

**Comment by xtrime-ru** [comment, marked as answer]
Вот инструкция по настройке на кинетике: https://ntc.party/t/неофициальный-docker-контейнер-антизапрета/8839/294

Просто указываете AllowedIPs = 10.0.0.0/8 и через туннель пойдет только DNS и трафик до заблокированных ресурсов. Это есть в конфиге wireguard (откройте файл в любом редакторе, он очень короткий и понятный). 

Есть еще несколько диапазонов заблокированных по ip, которые в openvpn прилетают на клиент. Но это не критично.  Я их чуть позже добавлю и в конфиг WG.

**Reply by VitArts** [reply]
Спасибо!

**Reply by xtrime-ru** [reply]
Вот еще пояснение, зачем все так сложно: 

https://ntc.party/t/неофициальный-docker-контейнер-антизапрета/8839/304

```
Если же отключить получение адресов от провайдера вообще, то при падении туннеля (например, если сервер умрет или Роскомнадзор начнёт резать протокол) некому будет резолвить вообще.

Отсюда необходимость в принципе прописывать сторонние DNS и создавать роуты.
```

Т.е. если вам не критично, что бы DNS работал в случае падения туннеля (что бывает не очень часто), то можно и не делать ничего кроме отключения провайдерских DNS и базовой настройки WG.

**Reply by RazGame** [reply]
Привет!
Что я сделал:
сгенерировал пароль, поднял админку, скачал конфиг.
В конфиге: Сделал пустым значение DNS и Allowed IPs оставил только 10.0.0.0/8
В роутере: Загрузил конфиг, в профиль DNS Antizapret добавил
77.88.8.8; 9.9.9.10; 10.0.0.1; 8.8.8.8; 1.1.1.1 + IP своего сервера 
Маршруты добавил по инструкции по ссылке, только в route ADD 195.208.4.1 MASK 255.255.255.255 10.8.0.2 
заменил 195.208.4.1 на ip своего сервера ( не уверен, что это правильно)

Подключение поднимается на роутере, но при открытии ntc, например, получаю ошибку в браузере
Подключение не защищено
Возможно, злоумышленники пытаются похитить вашу информацию с сайта ntc.party
net::ERR_CERT_COMMON_NAME_INVALID


что я делаю не так?
